### PR TITLE
Fix: Ingest walk function and tests

### LINKED ISSUE #5094: Fix ingest walker and tests
Test are now skipped due to wrong configuration env var.

### PR DESCRIPTION
**Pull request by N-o-Z**
Closes #5094

### CODE REVIEW

**Comment by N-o-Z** [comment]
Test will pass after #5096 is merged

File: esti/ugc_test.go [hunk 1]
```diff
@@ -301,7 +300,8 @@ func listUnderlyingStorage(t *testing.T, ctx context.Context, s3Client *s3.S3, b
 	for _, obj := range listOutput.Contents {
 		objects = append(objects, Object{
 			Key:          fmt.Sprintf("s3://%s/%s", bucket, aws.StringValue(obj.Key)),
-			LastModified: *obj.LastModified})
+			LastModified: *obj.LastModified,
+		})
```
Comment (nopcoder) [inline esti/ugc_test.go:304]: Just to be on the safe side
```suggestion
			LastModified: aws.TimeValue(obj.LastModified),
		})		
```

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: esti/ugc_test.go (hunk 1 above)
Comment (nopcoder) [reply esti/ugc_test.go:304]: without the space padding

**Review by nopcoder — APPROVED** [review]
LGTM!

**Pull request merged**